lib/misty/http/net_http.rb

```diff
@@ -3,6 +3,10 @@ module HTTP
     module NetHTTP
       def net_http(uri, ssl_verify_mode, log)
         http = Net::HTTP.new(uri.host, uri.port)
```

Comment:
I am not sure that makes sense because you have to declare in any case the http variable outside the if/else block